packages/plugin-types/src/ssm_environment_entries_generator.ts

```diff
@@ -0,0 +1,9 @@
+import { Construct } from 'constructs';
+import { SsmEnvironmentEntry } from './resource_access_acceptor.js';
+
+export type SsmEnvironmentEntriesGenerator = {
```

Comment:
This type defines a new property that is attached to the context object passed into `ConstructFactory.getInstance()`. It allows the construct factory to creates SSM references for properties without having to know anything about SSM or our naming conventions

Reply:
Refactored this to now be passed into GenerateContainerEntry